app/src/main/java/org/oppia/android/app/help/HelpActivity.kt

```diff
@@ -35,6 +35,9 @@ class HelpActivity :
   LoadThirdPartyDependencyListFragmentListener,
   LoadLicenseListFragmentListener,
   LoadLicenseTextViewerFragmentListener {
+  override fun onBackPressed() {
```

Comment:
Add blank line above

Reply:
Also I don't this function is needed as nothing is implemented here.

Reply:
done